### PR TITLE
Add support for ix:continuation

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This adds support for facts with content split across multiple blocks with ix:continuation.

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: iXBRLViewerPlugin/viewer/src/js/viewer.js [hunk 1]
```diff
@@ -66,11 +85,21 @@ Viewer.prototype._preProcessiXBRL = function(n, inHidden) {
         node = $(n).parent();
     }
     node.addClass("ixbrl-element").data('ivid',n.getAttribute("id"));
+    if (n.getAttribute("continuedAt")) {
+        node.data('iv-continued-at', n.getAttribute("continuedAt"));
```
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/viewer.js]: How is this `iv-continued-at` used?

File: samples/src/continuation/continuation.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (derekgengenbacher-wf) [inline samples/src/continuation/continuation.html]: Do we need all these extra lines?

File: iXBRLViewerPlugin/viewer/src/js/inspector.js [hunk 3]
```diff
@@ -307,7 +307,7 @@ Inspector.prototype.update = function () {
         var a = new Accordian({
             onSelect: function (id) { 
                 inspector._currentFact = inspector._report.getFactById(id)
-                inspector._viewer.highlightFact(inspector._currentFact); 
+                inspector._viewer.highlightFact(inspector._currentFact.id);
```
Comment (danaoredson-wf) [inline iXBRLViewerPlugin/viewer/src/js/inspector.js]: If this line just used `id`, like so, could line 309 just be removed?

```
inspector._viewer.highlightFact(id); 
```

Comment (brettkail-wk) [reply iXBRLViewerPlugin/viewer/src/js/inspector.js]: I don't think so, at least not trivially.  `inspector.update()` on line 311 uses `inspector._currentFact` to update calculations, references, URL fragment, etc.

File: iXBRLViewerPlugin/viewer/src/js/viewer.js (hunk 1 above)
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/viewer.js]: It's not.  A relic from a previous approach.  Removed in 9aef18b.

File: samples/src/continuation/continuation.html (hunk 2 above)
Comment (paulwarren-wk) [reply samples/src/continuation/continuation.html]: The files are generated using a templating system that's a bit too generous with its whitespace.  I've run it through a pretty printer in 8f96941 .  At some point I'll get the templater wired into the sample builder, but it's not quite ready yet.

File: iXBRLViewerPlugin/viewer/src/js/inspector.js (hunk 3 above)
Comment (danaoredson-wf) [reply iXBRLViewerPlugin/viewer/src/js/inspector.js]: 👍

Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/inspector.js]: Yes - assigning to this._currentFact changes the currently selected fact when the inspector is updated, but on reflection, this is messy.  I've refactored this in c55557c so we've now got Inspector.selectFact() which changes the current fact, and the current list of nested facts, and Inspector.switchFact() (called by selectFact()) which changes within the current list.

I've also refactored how the viewer interacts with the inspector as we had multiple routes to getting facts highlighted within the viewer.

**Comment by derekgengenbacher-wf** [comment]
QA +10 Unit tests pass successfully. Builds successfully. Locally smoke tested successfully. Verified that highlighting for facts with continuations work correctly.

@Workiva/release-management-p

**Pull request merged**